### PR TITLE
color-scheming: remove $border-ultra-light-gray

### PR DESCRIPTION
**Pull request by flootr**
#### Changes proposed in this Pull Request

* Replace usages of `$border-ultra-light-gray` with `--color-border-subtle`

#### Testing instructions

* Make sure each assignment is correct and there are no typos
* Check grays are applied as stated in https://github.com/Automattic/wp-calypso/pull/30607#issuecomment-461577898

### CODE REVIEW

**Comment by flootr** [comment]
@drw158 I'm not sure about introducing another layer of border colors. What do you think about using `--color-border-subtle` instead of introducing `--color-border-ultra-subtle`? This would mean visual changes where as right now there aren't any visual changes as it's the same underlying color (`$muriel-gray-0`).

**Comment by sixhours** [comment]
TL;DR: I went through the changes to find some visual examples of what might happen if we did away with `--border-color-ultra-subtle` in favor of `--border-color-subtle` and I think it's safe to make that change.

In a couple cases, the use case has no contrast with the background anyway:

Using `--border-color-ultra-subtle` in the sorted grid media library (no contrast with the background):

screen shot 2019-02-07 at 2 52 55 pm

The sorted grid media library with `--border-color-subtle` instead:

screen shot 2019-02-07 at 2 51 43 pm

If we don't like the lines after each section's date, we can remove that element rather than using a different border color.

Same idea with the border around new application passwords; this uses `--border-color-ultra-subtle`:

screen shot 2019-02-07 at 2 55 01 pm

And here it is with `--border-color-subtle`:
screen shot 2019-02-07 at 2 55 28 pm

We can just remove the border if we don't want to show it.

It's used in a few places in the WooCommerce app which need to be tested. One WC component uses it around the location flags:

screen shot 2019-02-07 at 3 00 50 pm

I think the `--border-color-subtle` looks better in this case:

screen shot 2019-02-07 at 3 01 09 pm

**Comment by blowery** [comment]
@flootr +1 to just using the existing subtle variable. I think it looks fine.

**Comment by drw158** [comment]
Agreed with all of the above.

**Comment by flootr** [comment]
@drw158 @blowery @sixhours made requested changes, another look would be highly appreciated 🙏
